openfisca_france/tests/formulas/irpp.yaml

```diff
@@ -952,3 +952,20 @@
     f4ba: 150000
   output_variables:
     irpp: -48036
+- name: "IRPP - Célibataire ayant des 	Revenu net imposable de 16000 € (teste la décote)"
+  period: 2014
+  absolute_error_margin: 0.9 #TODO : check why the error margin is so large, is there bottom round made by dgfip ?
+  input_variables:
+    salaire_imposable: 16000
+  output_variables:
+    irpp: -183
+    decote: 476
+#TODO : check where the 24.5 difference in IRPP comes from.
```

Comment:
> Since your are not supposed to commit a test that do not pass, I commented it.

@adrienpacifico Thanks to ask!

Tests must be valid in order to be merged into `next`, but since you are on your branch you can leave invalid tests.

For me these invalid tests an indicator that I must not merge the pull-request :)

Once you fix them, the pull-request will launch Travis tests again automatically. Did you know that if you push new commits in the pull-request branch (ie `decote_2015`), the pull-request updates itself?